packages/cli/src/commands/update.ts

```diff
@@ -49,7 +50,7 @@ const register: (program: any) => any = (program) => program
 
 async function action(contractReference: string, options: any): Promise<void> {
   const { force, interactive, all } = options;
-  const networkOpts = await promptForNetwork(options);
+  const networkOpts = await promptForNetwork(options, () => getCommandProps());
```

Comment:
Why this change?

Reply:
Because `promptForNetwork` has been moved to `prompt.ts` and it needs access to local `getCommandProps` function.